src/main/java/pl/byczazagroda/trackexpensesappbackend/service/HelloWorldService.java

```diff
@@ -12,4 +12,6 @@ public String sayHello() {
         return "Hello World!";
     }
 
+    public void deleteSingleWallet(Long idWallet) {
```

Comment:
This method should not be located in this service.

Reply:
this has been corrected -> service method is located in proper service